config/database.yml

```diff
@@ -0,0 +1,33 @@
+# SQLite. Versions 3.8.0 and up are supported.
+#   gem install sqlite3
+#
+#   Ensure the SQLite 3 gem is defined in your Gemfile
+#   gem "sqlite3"
+#
+default: &default
+  adapter: postgresql
+  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  timeout: 5000
+
+development:
+  <<: *default
+  database: MyDatabase
+  encoding: unicode
+  host: localhost
+  pool: 5
+  username: postgres
+  password: admin
```

Comment:
- [ ] **Optional:** In future projects, especially real apps, it is conventional to use `ENV` - environmental variables to hold your username and password. For development, the username and/or password for another developer varies from yours and using ENV ensures that any other developer can use your app without manually tempering your setup. [See this video](https://youtu.be/O-aDLsuNTRY) and do a little google search around this topic